baseplate/sidecars/live_data_watcher.py

```diff
@@ -145,16 +167,15 @@ def _load_from_s3(data: bytes) -> bytes:
         # a public resource belonging to another cluster/AWS account unless the request credentials
         # are unsigned.
 
-        # Access S3 with 10 max retries enabled:
+        # Default # of retries in legacy mode (current mode) is 5.
         s3_client = boto3.client(
             "s3",
-            config=Config(signature_version=UNSIGNED, retries={"total_max_attempts": 10}),
+            config=Config(signature_version=UNSIGNED),
             region_name=region_name,
         )
```

Comment:
Per https://reddit.slack.com/archives/C067Q6N14LA/p1701373678708859?thread_ts=1701370864.667409&cid=C067Q6N14LA we want to revert the changes made in [PR/813](https://github.com/reddit/baseplate.py/pull/813) where we increased retry total_max_attempt from 5->10

cc @areitz @jerroydmoore